packages/client/src/users/getUser.js

```diff
@@ -1,10 +1,10 @@
-import client, { adaptError } from '../../helpers/client';
+import client, { adaptError } from '../helpers/client';
 
 /**
  * Method responsible for fetching the logged user data.
  *
- * @function getProfile
- * @memberof module:profile/client
+ * @function getUser
+ * @memberof module:UserSessionExpiredError/client
```

Comment:
Just to double-check, is this correct?

Reply:
Nope, fixed, reviewed the other ones just to be assured